packages/driver/cypress/integration/e2e/multi-domain/basic_login_spec.ts

```diff
@@ -94,8 +93,10 @@ describe('basic login', { experimentalSessionSupport: true }, () => {
         cy.visit('http://www.foobar.com:3500/fixtures/auth/index.html')
       })
 
+      // @ts-ignore
       // TODO: Add createDomain primary domain config does not exist yet
       it.skip('D logs in with primary set via switch to domain config', { baseUrl: undefined, primaryDomain: 'localhost' }, () => {
+        // @ts-ignore
         cy.createDomain('http://idp.com:3500', () => { // PrimaryDomain set to localhost
```

Comment:
I should clean up this test file in another PR. I could probably either delete this test or switch it over to cy.origin